shell/gpu/gpu_surface_metal.mm

```diff
@@ -39,32 +39,25 @@
 }
 
 // |Surface|
-std::unique_ptr<SurfaceFrame> GPUSurfaceMetal::AcquireFrame(const SkISize& size) {
+std::unique_ptr<SurfaceFrame> GPUSurfaceMetal::AcquireFrame(const SkISize& frame_size) {
   if (!IsValid()) {
     FML_LOG(ERROR) << "Metal surface was invalid.";
     return nullptr;
   }
 
-  if (size.isEmpty()) {
+  if (frame_size.isEmpty()) {
     FML_LOG(ERROR) << "Metal surface was asked for an empty frame.";
     return nullptr;
   }
 
-  const auto bounds = layer_.get().bounds.size;
-  const auto scale = layer_.get().contentsScale;
-  if (bounds.width <= 0.0 || bounds.height <= 0.0 || scale <= 0.0) {
-    FML_LOG(ERROR) << "Metal layer bounds were invalid.";
-    return nullptr;
-  }
```

Comment:
Why is this not needed anymore?  Was it just wrong?

Reply:
This was redundant. The frame_size already accounts for these calculation. There is no point in doing them again here.